src/frontend/turbo.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://turbo.build/schema.json",
+    "pipeline": {
+        "build": {
+            "dependsOn": ["^build"],
+            "outputs": ["dist/**"]
+        },
+        "test": {
+            "outputs": []
+        },
+        "dev": {
+            "dependsOn": ["^dev"],
+            "cache": false
+        },
+        "extract-translations": {
+            "outputs": []
```

Comment:
Maybe could you configure a `outputs` options to setup a cache for this task, nope ?